docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -277,6 +277,18 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     min(minute),h(hours),d(days),w(weeks),m(months).
                                     If a period is not specified it defaults to
                                     days.
+      --spark-property <arg>        Filter applications based on certain Spark properties that were
+                                    set during launch of the application. It can filter based
+                                    on key:value pair or just based on keys. Multiple configs
+                                    can be provided where the filtering is done if any of the
+                                    config is present in the eventlog.
+                                    filter on specific configuration(key:value):
```

Comment:
does the : work with spark.master = spark://5.6.7.8:7077  ?

What about regex like spark.redaction.regex or spark.eventlog.dir (file:///tmp/spark-events)

This is why I want more tests, go through configs at http://spark.apache.org/docs/latest/configuration.html and look for various formats

Reply:
Thanks for the providing the link.  Went through the list to determine different formats and added tests. Please let me know if you would like to see more tests.  For spark.redaction.regex it seems we can filter only based on keys as the values are redacted.